components/panel/src/PanelBody.js

```diff
@@ -2,23 +2,16 @@
 
 import React from 'react';
 import { jsx, useTheme, paint } from '@westpac/core';
-import { propTypes, defaultProps } from './Panel';
 
 // ==============================
 // Component
 // ==============================
 
-export const PanelBody = ({ appearance, ...props }) => {
-	const { breakpoints, panel } = useTheme();
+export const PanelBody = props => {
+	const {
+		LAYOUT: { breakpoints },
+	} = useTheme();
 	const mq = paint(breakpoints);
```

Comment:
Not directly relate to this PR, but it seems odd that we're passing `paint` around and creating the helper utility inside each component. Might be a good time to abstract it e.g.

```jsx
// Somewhere in Core?
export const useMediaQuery = () => {
  const { LAYOUT } = useTheme();

  return paint(LAYOUT.breakpoints);
}

// Use
export const Well = props => {
  const mq = useMediaQuery();

  return (
    <div css={mq({
      query: 'reliant',
      styles: ['can', 'go', 'here']
    })} {...props} />
  );
}
```

Reply:
implemented in separate pr :) 
https://github.com/WestpacGEL/GEL/pull/141